Pacman Pygame/leveleditor.py

```diff
@@ -51,19 +51,9 @@
 	for event in pygame.event.get():
 		if event.type == QUIT:
 			exit = True
-	if event.type == pygame.KEYDOWN:
-		#print(str(event.key))
-		if event.key == pygame.K_LEFT:
-			x += -20
-			#print(str(x)+", "+str(y))
-		elif event.key == pygame.K_RIGHT:
-			x += 20
-			#print(str(x)+", "+str(y))
-		elif event.key == pygame.K_UP:
-			y += -20
-			#print(str(x)+", "+str(y))
-		elif event.key == pygame.K_DOWN:
-			y += 20
+	if event.type == pygame.mouse.get_pressed()[0]:
+		x=pygame.mouse.get_pos()[0]
+		y=pygame.mouse.get_pos()[1]
 		if event.key == pygame.K_c:
 			if (value,x,y,rectwidth,rectheight) not in xy:
 				xy.append((value,x,y,rectwidth,rectheight))
```

Comment:
So I basically just fixed an indentation errors and one outdated method, and then added back in keyboard support but you can change the increment that you move on the keyboard with "p". So if you think that works together well overall then cool, or if you have a better idea thats cool too. 

I have the code pasted below, if you paste it under the "exit = True" line i think it should work, but you might need to mess with it a little 

and also make sure to declare these variables before the main loop: (also i just attached a txt file with the whole code incase copy/paste does not work. 
[mouse_support_revision.txt](https://github.com/migerovitch/Pacman-Project/files/3817455/mouse_support_revision.txt)



arrowIncrement = 0
arrowIncrements = [2, 10, 20] 

Alex

if event.type == pygame.MOUSEBUTTONDOWN:
			if pygame.mouse.get_pos()[0]%2 == 0:
				x=pygame.mouse.get_pos()[0]
				y=pygame.mouse.get_pos()[1]
			else:
				x=pygame.mouse.get_pos()[0] + 1
				y=pygame.mouse.get_pos()[1] + 1

		if event.type == pygame.KEYDOWN:
		#print(str(event.key))
			if event.key == pygame.K_p:
				if arrowIncrement == 2:
					arrowIncrement = 0
				else:
					arrowIncrement += 1
				while x%arrowIncrements[arrowIncrement] != 0:
					x += 1

				while y%arrowIncrements[arrowIncrement] != 0:
					y += 1

			if event.key == pygame.K_LEFT:
				x += -arrowIncrements[arrowIncrement]
				#print(str(x)+", "+str(y))
			elif event.key == pygame.K_RIGHT:
				x += arrowIncrements[arrowIncrement]
				#print(str(x)+", "+str(y))
			elif event.key == pygame.K_UP:
				y += -arrowIncrements[arrowIncrement]
				#print(str(x)+", "+str(y))
			elif event.key == pygame.K_DOWN:
				y += arrowIncrements[arrowIncrement]
			if event.key == pygame.K_c:
				if (value,x,y,rectwidth,rectheight) not in xy:
					xy.append((value,x,y,rectwidth,rectheight))
					print("Block saved to list")
			if event.key == pygame.K_r: